### PR TITLE
allow coefficient for additive coag kernel to be set from spec file

### LINKED ISSUE #186: allow setting the value of additive kernel constant from a spec file

### PR DESCRIPTION
**Pull request by zdaq12**
Resolves #186

### CODE REVIEW

**Review by jcurtis2 — COMMENTED** [review]
While the code looks good and accomplishes the purpose, I am wondering if `env_state_t` is the place for this variable `beta_1` to reside. While it was least disruptive to existing code to place there as the coagulation kernel functions already take the `env_state_t`, my possible alternative suggestion would be placing it in `scenario_t` (and then passing that structure around). I  think one could argue that this scaling factor is pretty similar in purpose to the wall loss parameters used for `chamber_t` which is a member of `scenario_t`.

@mwest1066 any thoughts?

**Comment by mwest1066** [comment]
Are we thinking of these parameters as being something point-wise in space that affects the coagulation kernel? E.g., something like a turbulent kinetic energy that might affect a coagulation kernel? If so, then I think that this is something which could vary in space and time, and so it does make sense to think of it as similar to a temperature or pressure, and hence to be included in `env_t`.

File: src/env_state.F90 [hunk 1]
```diff
@@ -73,6 +73,8 @@ module pmc_env_state
      real(kind=dp) :: solar_zenith_angle
      !> Box height (m).
      real(kind=dp) :: height
+     !> Scaling coefficient for additive coagulation kernel.
+     real(kind=dp) :: beta_1
```
Comment (mwest1066) [inline src/env_state.F90:77]: I don't like the name of this variable (it's very mysterious until you read the comment). Could we call it something like `additive_kernel_coefficient` instead?

Comment (jcurtis2) [reply src/env_state.F90]: Changed from `beta_1` to `additive_kernel_coefficient` throughout code in https://github.com/compdyn/partmc/pull/187/commits/5177a4646d60a7374f4e27dcc120c4d4da43da30

**Review by mwest1066 — APPROVED** [review]

**Pull request merged**